tests/checkers/unittest_imports.py

```diff
@@ -92,28 +93,33 @@ def test_relative_beyond_top_level_four(capsys: CaptureFixture[str]) -> None:
         assert errors == ""
 
     def test_wildcard_import_init(self) -> None:
-        module = astroid.MANAGER.ast_from_module_name("init_wildcard", REGR_DATA)
-        import_from = module.body[0]
+        context_file = os.path.join(REGR_DATA, "dummy_wildcard.py")
```

Comment:
Why is the context file necessary and doesn't it change what we're testing ?

Reply:
Context file would be used to specify that the file where the import statement for the module. I was trying to understand the intent of the code and the test and trying to guess what we are trying to achieve. The existing test only works if the search directory is in the `sys.path`. If we want this test to succeed without changing `sys.path`, then I think we need to change (fix?) the `find_spec` in `astroid.interpreter._import.spec` module. It is not considering the directory we are passing to it. Shall I attempt to push a fix and see if it works?